CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+If you'd like to contribute, the best approach is to send a well-formed pull
+request:
+- [ ] tested
+- [ ] including documentation
+
+[Pull requests](https://docs.github.com/en/pull-requests) should be focused:
+trying to do more than one thing in a single request will make it more
+difficult to process.
+
+If you have a bug or feature request you can try [logging an issue](/issues).
```

Comment:
This link should work, if I understand https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files correctly:

>  Links starting with / will be relative to the repository root.